manifests/j/JAMSoftware/TreeSize/9.2.1/JAMSoftware.TreeSize.installer.yaml

```diff
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.6.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: JAMSoftware.TreeSize
+PackageVersion: 9.2.1
+InstallerLocale: en-US
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+InstallerType: inno
+Scope: machine
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+UpgradeBehavior: install
+ProductCode: TreeSize_is1
+AppsAndFeaturesEntries:
+- DisplayName: TreeSize V9.0.3 (64 bit)

```

Comment:
```suggestion
```

This is incorrect for the 9.2.1 version, and DisplayName is strictly not needed for this manifest. Can you remove this please?